sdk/v1/client_test.go

```diff
@@ -679,3 +678,60 @@ func TestClient_GetTask(t *testing.T) {
 		})
 	})
 }
+
+func TestClient_GetJob(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+	getJobPath := "/v1/jobs/883c81fd-726d-4ea3-9db8-7e7c781a01cc"
+
+	reqHeaders := client.Headers{
+		IfMatch:          "*",
+		ServiceAuthToken: "",
+	}
+
+	Convey("Given clienter.Do doesn't return an error", t, func() {
+		body, err := json.Marshal(expectedJob)
+		if err != nil {
+			t.Errorf("failed to setup test data, error: %v", err)
+		}
+
+		httpClient := newMockHTTPClient(
+			&http.Response{
+				StatusCode: http.StatusCreated,
+				Body:       io.NopCloser(bytes.NewReader(body)),
+				Header: http.Header{
+					"Etag": []string{expectedTask.ETag},
+				},
+			},
+			nil)
+
+		searchReindexClient := newSearchReindexClient(t, httpClient)
+
+		Convey("When search-reindexClient.GetJob is called", func() {
+			respHeaders, job, err := searchReindexClient.GetJob(ctx, reqHeaders, testJobID)
+			So(err, ShouldBeNil)
+
+			Convey("Then the expected jobid, searchindexname, state, totalsearchdocument, and totalinsertedsearchdocuments of documents, are returned", func() {
+				So(job.ID, ShouldEqual, "883c81fd-726d-4ea3-9db8-7e7c781a01cc")
+				So(job.SearchIndexName, ShouldEqual, "ons123456789")
+				So(job.State, ShouldEqual, "created")
+				So(job.TotalSearchDocuments, ShouldEqual, 10)
+				So(job.TotalInsertedSearchDocuments, ShouldEqual, 5)
```

Comment:
In lines 715-719 it would be good to be clear that the values in the job are being compared to those in the expectedJob. E.g. 

So(job.ID, ShouldEqual, expectedJob.ID)
So(job.SearchIndexName, ShouldEqual, expectedJob.SearchIndexName)
So(job.State, ShouldEqual, expectedJob.State)
So(job.TotalSearchDocuments, ShouldEqual, expectedJob.TotalSearchDocuments)
So(job.TotalInsertedSearchDocuments, ShouldEqual, expectedJob.TotalInsertedSearchDocuments)